service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceController.kt

```diff
@@ -443,3 +436,17 @@ fun wasAbsentFromPreschoolDaycare(placementType: PlacementType, arrived: LocalTi
 
     return false
 }
+
+fun wasAbsentFromPaidDaycare(placementBasics: ChildPlacementBasics, arrived: LocalTime, departed: LocalTime): Boolean {
+    val (placementType, dateOfBirth) = placementBasics
+
+    if (!isEntitledToFreeFiveYearsOldDaycare(dateOfBirth)) {
+        return false
```

Comment:
Shouldn't this be `true` or am I missing something? I don't fully understand what's going on here.

Reply:
No, but the function name may still be misleading. The function is used when child is leaving to determine if we should add an absence for the paid portion of the daycare. That scenario only applies to 5yo case, hence false. Maybe I'll rename it as `wasAbsentFrom5yoPaidDaycare`?

Reply:
Yeah that rename sounds like a good idea.